Python/pywarpx/picmi.py

```diff
@@ -375,11 +375,16 @@ def initialize_inputs(self):
 
         # Geometry
         pywarpx.geometry.coord_sys = 1  # RZ
-        pywarpx.geometry.is_periodic = '0 %d'%(self.bc_zmin=='periodic')  # Is periodic?
         pywarpx.geometry.prob_lo = self.lower_bound  # physical domain
         pywarpx.geometry.prob_hi = self.upper_bound
         pywarpx.warpx.n_rz_azimuthal_modes = self.n_azimuthal_modes
 
+        # Boundary conditions
+        pywarpx.boundary.field_lo = '%s %s' % (self.bc_rmin, self.bc_zmin)
```

Comment:
The conversion of a list to a string will be handled automatically elsewhere so the formatting is not needed. Since some mapping of names should be done, I would suggest something like this `pywarpx.boundary.field_lo = [BC_map[bc] for bc in [self.bc_rmin, self.bc_zmin]]` where BC_map is a dictionary with entries like `{'open':'pml'}`.

Reply:
I've incorporated this approach for the field BC. Seeing as there are no conflicts in the naming conventions for particle boundary conditions between picmistandard and WarpX I've made the call for that simply
`pywarpx.boundary.particle_lo = [self.bc_rmin_particles, self.bc_zmin_particles]`
I thought this was best specifically because there is a collision between the field and particle boundary conditions - `open` for field should be `pec` but `open` for particles should be `open`.